md/configure-client-of-bonita-bpm-engine.md

```diff
@@ -42,9 +42,12 @@ You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB
 ::: warning
 We do not guarantee to keep the http protocol stable, so we strongly recommend that you use the same version for both
 the client and the server
+:::
+::: danger
+EJB communication protocol is removed in 7.10.
```

Comment:
```suggestion
EJB communication protocol will be removed in 7.10.
```

Reply:
I though having `is` is more direct and there will be no change once we release 7.10
Do we keep `is` or put `will be`?

Reply:
ok